stable/datadog/README.md

```diff
@@ -116,3 +118,11 @@ datadog:
           port: 6379
 ```
 
+### leader election
+
+The Datadog Agent supports built in leader election option for the Kubernetes event collector As of 5.17.
+This feature relies on ConfigMaps, enabling this flag will grant Datadog Agent get, list, delete and create access to the ConfigMap resource.
+See the full [RBAC](https://github.com/DataDog/integrations-core/tree/master/kubernetes#gathering-kubernetes-events)
+Agents coordinate by performing a leader election among members of the Datadog DaemonSet through kubernetes to ensure only one leader agent instance is gathering events at a given time.
```

Comment:
The transition here is a bit strange. Keep in mind there's no carriage return in markdown.
It would also be nice to add a warning in bold that these RBAC entities **will** need to be created before the option is set.